conda-recipes/scvi/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "scvi" %}
+{% set version = "0.1.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "ca87155662d85605f86c5e86b7b9f64d881b882177b9642fff8f0ea215c6cb1a" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pip >=9.0.1
+    - flake8 ==3.5.0
+    - coverage ==4.5.1
+    - sphinx ==1.7.1
+    - pytest ==3.4.2
+    - pytest-runner ==2.11.1
+    - numpy ==1.14.0
+    - torch ==0.4.0
+    - matplotlib ==2.2.2
+    - scikit-learn ==0.19.1
+    - scipy ==1.0.0
+    - h5py
+    - codecov
+    - pandas ==0.23.0
+    - loompy ==2.0.9
+  run:
+    - python
+    - pip >=9.0.1
+    - flake8 ==3.5.0
+    - coverage ==4.5.1
+    - sphinx ==1.7.1
+    - pytest ==3.4.2
+    - pytest-runner ==2.11.1
+    - numpy ==1.14.0
+    - torch ==0.4
+    - matplotlib ==2.2.2
+    - scikit-learn ==0.19.1
+    - scipy ==1.0.0
+    - h5py
+    - codecov
+    - pandas ==0.23.0
+    - loompy ==2.0.9
+
+test:
+  imports:
+    - scvi
+    - scvi.dataset
+    - scvi.metrics
+    - scvi.models
+  requires:
+    - pytest
```

Comment:
Does this run our tests? How do you decide what to put in `imports`?

Reply:
I found a documentation on [building conda from pip package](https://conda.io/docs/user-guide/tutorials/build-pkgs-skeleton.html). The recipes are automatically generated from `pip` package. It words when I tried building `pyinstrument` from `pip`, but I got the `pytorch` error when I tried building `scvi`. :(

Reply:
I see. Actually `pytorch` exists in both the default channel and the pytorch channel, so I'm not surprised this didn't help.

Reply:
You might post the question on stackoverflow. Seems like there's a got to be a way....

Reply:
Okay. 👌🏻